docs/workflow/testing/coreclr/unix-test-instructions.md

```diff
@@ -74,16 +74,23 @@ Is used to disable the build, that way if building on Unix cycles are saved buil
 PAL tests
 ---------
 
-Build CoreCLR on the Unix machine.
+Build CoreCLR with PAL tests on the Unix machine:
+
+```sh
+cd runtime
+./src/coreclr/build-runtime.sh -skipgenerateversion -nopgooptimize \
+    -cmakeargs -DCLR_CMAKE_BUILD_TESTS=1
```

Comment:
@janvorli, @jkotas, imo, in the future it would be nice if we add an explicit option in `build-runtime.sh` to build PAL tests or move these tests under `src/coreclr/tests` with option in `build-test.sh` to skip everything, and only build PAL tests for new platforms (where `dotnet(1)`/msbuild is not available yet).

ps: i am also looking into fixing build breakage on macOS, due of unavailability of `<uchar.h>` header (for char16_h definition).

Reply:
We used to run PAL tests on every PR, somehow this got broken in the past and it is not clean to me when.

Reply:
@janvorli, is the doc change good to go?

Reply:
I have opened https://github.com/dotnet/runtime/issues/34776 to continue this discussion.